__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const app = require('../app')
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data/index')
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+    test('GET 200 - responds with array of topic objects, with slug and description properties', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            const { topics } = body
+            expect(Array.isArray(topics)).toBe(true)
+            expect(topics).toHaveLength(3)
+            topics.forEach((topic) => {
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+            })
+        })
+    })
+    test('Error 404 - path not found', () => {
```

Comment:
We can pop this test in a separate describe block, as it's going to be relevant to the whole server and not just api/topics 👍

Reply:
Completed